home/tests/output.json

```diff
@@ -0,0 +1 @@
+{"articles": [["article title", "article subtitle", ["<p>this is some plain text</p>", "<p>this is some richtext</p>"], ["tag1", "tag2"]]]}
```

Comment:
shouldn't each element of the articles list be a dict, rather than a list?
Then ordering of the title, subtitle, ect won't matter

Reply:
will change